lib/cocoapods/installer.rb

```diff
@@ -163,6 +163,7 @@ def install!
       validate_targets
       if installation_options.skip_pods_project_generation?
         show_skip_pods_project_generation_message
+        run_podfile_post_install_hooks
```

Comment:
Makes sense to me. Spec for this can be adapted from this existing one https://github.com/CocoaPods/CocoaPods/blob/6896aa979505035a6f2a61fa1583a89952388d99/spec/unit/installer_spec.rb#L95-L118

Reply:
thx for looking into this! going to resume pushing on this by writing the spec to get it red then green with this change